js/src/main.js

```diff
@@ -34,26 +51,46 @@ export default class MainApplication extends Component {
         <Redirect from='/' to='/accounts' />
         <Redirect from='/auth' to='/accounts' query={ {} } />
         <Redirect from='/settings' to='/settings/views' />
+
+        { /** Backward Compatible links */ }
+        <Route path='/account/:address' onEnter={ this.handleDeprecatedRoute } />
+        <Route path='/address/:address' onEnter={ this.handleDeprecatedRoute } />
+        <Route path='/contract/:address' onEnter={ this.handleDeprecatedRoute } />
+
         <Route path='/' component={ Application }>
-          <Route path='accounts' component={ Accounts } />
-          <Route path='account/:address' component={ Account } />
-          <Route path='wallet/:address' component={ Wallet } />
-          <Route path='addresses' component={ Addresses } />
-          <Route path='address/:address' component={ Address } />
+          <Route path='accounts'>
+            <IndexRoute component={ Accounts } />
+            <Route path=':address' component={ Account } />
+            <Route path='/wallet/:address' component={ Wallet } />
+          </Route>
+
+          <Route path='addresses'>
+            <IndexRoute component={ Addresses } />
+            <Route path=':address' component={ Address } />
+          </Route>
+
           <Route path='apps' component={ Dapps } />
           <Route path='app/:id' component={ Dapp } />
-          <Route path='contracts' component={ Contracts } />
-          <Route path='contracts/write' component={ WriteContract } />
-          <Route path='contract/:address' component={ Contract } />
+
+          <Route path='contracts'>
+            <IndexRoute component={ Contracts } />
+            <Route path='write' component={ WriteContract } />
```

Comment:
While we are renaming and fixing up - the "write contract" is very confusing. Had 2 people asking me now where the find the editor. 

"Develop Contract" ?